openapi/paths/billing-portals.yaml

```
@@ -6,8 +6,8 @@ get:
   summary: Retrieve billing portals
   operationId: GetBillingPortalCollection
   x-sdk-operation-name: getAll
-  description: |
-    Retrieve a list of billing portals.
+  description: |-
+    Retrieves a list of billing portals.
```

```suggestion
  description: Retrieves a list of billing portals.
```
In this case of a one-liner, the block syntax isn't needed.